.travis/tooling.py

```diff
@@ -94,6 +94,22 @@ def affects_tests(path, task):
             print("+++ %s is definitely part of the %s tests" % (path, task))
             return True
 
+    # A number of Sierra-related tasks share code/Makefiles in the
+    # sierra_adapter directory.  If we're definitely in a project which
+    # has nothing to do with Sierra, we can ignore changes in this dir.
+    sierra_free_tasks = (
```

Comment:
👍  but is this because some things are in the wrong place?

Reply:
Not sure I understand the question?